nselib/tls.lua

```diff
@@ -195,6 +195,9 @@ ELLIPTIC_CURVES = {
   ffdhe4096 = 0x0102, --RFC7919
   ffdhe6144 = 0x0103, --RFC7919
   ffdhe8192 = 0x0104, --RFC7919
+  MLKEM512 = 0x0512 -- draft-connolly-tls-mlkem-key-agreement-03
```

Comment:
These are not the numbers actually assigned.
<img width="922" alt="Screenshot 2024-12-05 at 12 23 18" src="https://github.com/user-attachments/assets/da4d2b1d-f522-4aad-b260-eefe8be03d92">
https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml